docs/Project-Configuration/Parameterized-Builds.md

```diff
@@ -89,6 +99,11 @@ Can parameters be changed after a patch is finalized?
 
 * No, parameters are **not configurable** after the patch is finalized (i.e. after Schedule Patch is pressed, or `--finalize` is used when creating the patch from the CLI). Dynamically modifying parameters in a patch could affect task runtime and result in difficulty reproducing task behavior.
 
+Can I define separate parameters for PR patches?
+
+* No, it's only possible to configure parameters from the CLI or the configure page, 
+but PR patches are finalized immediately, so this latter option isn't available.
```

Comment:
This sentence sounds confusing to me, does the latter option mean the configure page? I think If you separate out the sentences like `configure page. PR patches are... so the latter option isn't available` it sounds better (but I might be wrong, english is hard)